src/odemis/util/dataio.py

```diff
@@ -65,12 +65,11 @@ def data_to_static_streams(data):
              (ci >= 0 and d.shape[ci] > 1)) or
             (ci >= 0 and d.shape[ci] >= 5)  # No metadata, but looks like a spectrum
            ):
-            if (MD_TIME_LIST in d.metadata and
-                (ti >= 0 and d.shape[ti] > 1)):
+            if (MD_TIME_LIST in d.metadata and (ti >= 0 and d.shape[ti] > 1)):
                 # Streak camera data. Create a temporal spectrum
                 name = d.metadata.get(model.MD_DESCRIPTION, "Temporal Spectrum")
                 klass = stream.StaticSpectrumStream
-            else:
+            else:  # TODO this is also temporal spectrum data acquired with mirror and focus mode (so no time/wl info)
```

Comment:
Indeed, we might want to drop the check for the presence of MD_TIME_LIST and/or MD_WL_LIST. Anyway, when we have data of CT1YX, I guess the temporal-spectrum views are the best to use.

Reply:
Could we not drop it now? It looks like the only difference is the MD_DESCRIPTION...

Reply:
Should I implement it in a separate pr or leave as it is for now?

Reply:
Leave this as-is, with the TODO.